### PR TITLE
Replace try with safe navigation operator

### PR DESCRIPTION
**Pull request by jacobherrington**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description

⌘ + ⇧ + F

Safe navigation is faster than `try`, so in my opinion, it's better to avoid any instance of `try` if possible.

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/models/tweet.rb [hunk 1]
```diff
@@ -73,7 +73,7 @@ def make_tweet_from_api_object(tweeted)
       tweet = Tweet.where(twitter_id_code: tweeted.attrs[:id_str]).first_or_initialize
       tweet.twitter_uid = tweeted.user.id.to_s
       tweet.twitter_username = tweeted.user.screen_name.downcase
-      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name).try(:id)
+      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name).&(:id)
```
Comment (maestromac) [inline app/models/tweet.rb]: actually, that would raise `NoMethodError: undefined method `&' for #<User:....>`

It should be the following
```suggestion
      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name)&.id
```

Comment (jacobherrington) [reply app/models/tweet.rb]: This tells me two things:
1) I am not thorough enough.
2) Automated tests need some love here.

😅 Thanks.

Comment (maestromac) [reply app/models/tweet.rb]: I nearly missed it too. This section is definitely missing test.

File: app/models/tweet.rb [hunk 2]
```diff
@@ -73,7 +73,7 @@ def make_tweet_from_api_object(tweeted)
       tweet = Tweet.where(twitter_id_code: tweeted.attrs[:id_str]).first_or_initialize
       tweet.twitter_uid = tweeted.user.id.to_s
       tweet.twitter_username = tweeted.user.screen_name.downcase
-      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name).try(:id)
+      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name)&.(:id)
```
Comment (lightalloy) [inline app/models/tweet.rb]: Btw it can be a bit shorter with `User.find_by(twitter_username: tweeted.user.screen_name)&.id`

**Comment by maestromac** [comment]
Since this is basically there I'm going to shorten the syntax a bit

**Pull request merged**